src/routes/guilds/guild/leaderboard.ts

```diff
@@ -0,0 +1,80 @@
+import { Route, RouteStore } from 'klasa-dashboard-hooks';
+import ApiRequest from '../../../lib/structures/api/ApiRequest';
+import ApiResponse from '../../../lib/structures/api/ApiResponse';
+import { Snowflake } from 'discord.js';
+import { LeaderboardUser } from '../../../lib/util/Leaderboard';
+
+export default class extends Route {
+
+	public constructor(store: RouteStore, file: string[], directory: string) {
+		super(store, file, directory, { route: 'guilds/:guild/leaderboard' });
+	}
+
+	public async get(request: ApiRequest, response: ApiResponse) {
+
+		const guildID = request.params.guild;
+		if (!this.client.guilds.has(guildID)) return response.error(400);
+		const lb = await this.client.leaderboard.fetch(guildID);
+
+		if ((Object.keys(request.query)).includes('id')) {
+
+			const id = request.query.id as string;
+			const lbData = lb.get(id);
+			if (!lbData) return response.error(400);
+
+			lbData.name = lbData.name ? lbData.name : (await this.client.users.fetch(id)).username;
```

Comment:
nit: Avoid useless assignments:
```suggestion
			if (lbData.name === null) lbData.name = (await this.client.users.fetch(id)).username;
```